ci-operator/step-registry/cucushift/upgrade/vsphere-hw/cucushift-upgrade-vsphere-hw-commands.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -f "${SHARED_DIR}/kubeconfig" ]] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
+fi
+
+# Setup proxy if it's present in the shared dir
+if [[ -f "${SHARED_DIR}/proxy-conf.sh" ]] 
+then
+    # shellcheck disable=SC1091
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+source "${SHARED_DIR}/govc.sh"
+# list all the virtual machines in the folder/rp
+cluster_name=$(<"${SHARED_DIR}"/clustername.txt)
+vm_path="/${GOVC_DATACENTER}/vm/${cluster_name}"
+clustervms=$(govc ls "${vm_path}-*")
```

Comment:
`${vm_path}` is correct folder name, no need to append `-*`.

Reply:
Updated. Thanks!